### PR TITLE
quickstart-short

### PR DESCRIPTION
**Pull request by hnjolles1**
Shortened version of the Quickstart guide
re:  pr 172

### CODE REVIEW

File: quickstart-short.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+### Considerations 
+
+To begin with, we will be running on the Arbitrum Rinkeby testnet. You will need to acquire token to be able to get started.
+
+- [Connect a Wallet](/video-miners/how-to-guides/l2-migration#migrating-to-arbitrum)
+
+    - Arbitrum Rinkeby testnet, Rinkeby ETH is made available free of charge.
+
+    - On Arbitrum (LP2 mainnet) you will need to purchase ETH to cover gas fees and LPT to stake and activate on mainnet. (See ETH and LPT)
```
Comment (hthillman) [inline quickstart-short.md:73]: So i'm trying to follow it as a totally naive individual and i got lost a little bit in this section.

I think "Connect a wallet" is a confusing call to action - I'm not sure whether it's a step or a section header

File: quickstart-short.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+**For Example** Livepeer Binary should be saved at a location of your choice and includes:
+
+    - `livepeer.exe`
+    - `livepeer_bench.exe`
+    - `livepeer_cli.exe`
+    - `livepeer_router.exe`
+
+### Navigate the Livepeer Explorer
+
+Slots available for delegating are limited to 100 orchestrators. View the top 100 Orchestrators to know the minimum amount staked to Activate on the network. You can visualize current Orchestrators and performance on the Performance Leaderboard.
```
Comment (hthillman) [inline quickstart-short.md:93]: For consistency in terminology:

`Slots available for delegating are limited to 100 orchestrators.`
->
`The network currently allows 100 orchestrators to be active and receive work.`

File: quickstart-short.md [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+- [Transcoder](/video-miners/getting-started/choosing-a-role#transcoder)
+
+### Download and [Install](/installation/livepeer-quickstart-shortversion#installation-workflow) `livepeer`
+
+**For Example** Livepeer Binary should be saved at a location of your choice and includes:
+
+    - `livepeer.exe`
+    - `livepeer_bench.exe`
+    - `livepeer_cli.exe`
+    - `livepeer_router.exe`
```
Comment (hthillman) [inline quickstart-short.md:89]: should we just sent people to the install link? i got a little confused here as well

File: quickstart-short.md [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+
+- Be sure to set the `-nvidia` GPU flag or the benchmarking will default to CPU transcoding and significantly slow down and impact the performance of your machine.
+
+-  You can set the `-concurrentSessions all` to transcode all the test segments or the number of segments you want to test.
+
+- Run the benchmarking tool `livepeer_bench.exe` 
+
+**Realtime Duration Ratio**: should be under 0.2 for proper performance and not lose any segments. 
+
+- Provide the number of `concurrentSessions` until your reach 0.2 so that streams can process at real-time speed--sufficient performance for participating on the network.
```
Comment (hthillman) [inline quickstart-short.md:123]: typo: `your` -> `you`

File: quickstart-short.md [hunk 5]
```diff
@@ -0,0 +1,273 @@
...
+On your router:
+- Set up a static IP Address/hostname
+- Set up your port forwarding with the service URI to port `8935`
+
+> **Note** Each router is distinct and in some cases, ports are hardcoded and do not allow for other configurations.
+
+
+### Set up your Network Connection
+
+ You will first run on the [Rinkeby Testnet](/video-miners/getting-started/testing/testnet).
```
Comment (hthillman) [inline quickstart-short.md:162]: `Rinkeby Testnet` -> `Arbitrum Rinkeby Testnet`

File: quickstart-short.md [hunk 6]
```diff
@@ -0,0 +1,273 @@
...
+
+You will be able to Acquire ETH via the faucet to send to Arbitrum via [Arbitrum Bridge](https://bridge.arbitrum.io/).
+
+You can Get the Arbitrum Rinkeby RPC Url via either an:
+
+- Offchain public testnet endpoint, or
+- Third-party service (Alchemy or Infuria) where you can set up an account with an Arbitrum Rinkeby RPC URL. Be sure to copy the URL and include it for your testnet setup.
```
Comment (hthillman) [inline quickstart-short.md:169]: `Infura`

File: quickstart-short.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+## What you need to know
+
+- familiarity with platform capabilities highlighted in [Protocol Core Concepts](/protocol/core-concepts/).
+- how to work with a command-line interface
+    -  `wget` is recommended (alternatively, `cURL`  can be used)
+    - [grpcurl](https://github.com/fullstorydev/grpcurl#grpcurl) for interacting with gRPC servers for testing and monitoring
+- how to perform installations with Binaries and/or Docker 
+- familiarity with networking concepts and configuration (URLs, ports and port forwarding, static IPs, )
+- ability to access an [EVM-compatible network](/installation/connect-to-ethereum)
+- working with faucets and funding wallets
```
Comment (hthillman) [inline quickstart-short.md:19]: I'd combine these two:

```
Understanding of how to send and receive funds using EVM-based networks
```

File: quickstart-short.md [hunk 8]
```diff
@@ -0,0 +1,273 @@
...
+---
+
+This Quickstart includes a detailed overview of how Livepeer works with contextual information about the steps to set up a [combined Livepeer Orchestrator and Transcoder](/video-miners/core-concepts/architecture).
+
+[**Video Miners**](/installation/quickstart#start-video-mining) will be able to install and set up an Orchestrator for video mining and use your current GPU for transcoding. 
+ 
+[**Developers**](/installation/quickstart#start-developing) will be able to install and setup Livepeer for developing.
+
+## What you need to know
+
```
[the remark this answers is not in the document]
Comment (hnjolles1) [reply quickstart-short.md:11]: if it is not a complete sentence, not necessarily -- it's a stylistic choice. i can change them if you like.

File: quickstart-short.md [hunk 9]
```diff
@@ -0,0 +1,273 @@
...
+- Git
+- Building from source, binaries, and Docker    
+
+## Technical Requirements
+
+Livepeer supports the following to install and run `livepeer` on the network:  
+
+**Operating Systems** 
+
+Linux, Darwin (macOS), and Windows
```
[the remark this answers is not in the document]
Comment (hnjolles1) [reply quickstart-short.md:27]: well, we are providing all of the operating systems it runs on, so i left it as an "and" but i can change it if you like.

File: quickstart-short.md [hunk 10]
```diff
@@ -0,0 +1,273 @@
...
+**Current Releases** for supported operating systems are found on our Github repository [releases page](https://github.com/livepeer/go-livepeer/releases).
+
+Depending on your workflow, you can install Livepeer:
+> * [Using a binary release](/installation/install-livepeer/binary-release)
+> * [Build from a Docker image](/installation/install-livepeer/docker)
+> * [Perform a Developer Install](/installation/install-livepeer/installing-for-development)
+
+## Start Video Mining
+
+You will be setting up a standalone Orchestrator with a transcoder so that you will be able to manage it, get it listed on the Orchestrator list and view it on the Livepeer Explorer.
```
[the remark this answers is not in the document]
Comment (hnjolles1) [reply quickstart-short.md:53]: definitely, thanks for the good catch.

File: quickstart-short.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (hthillman) [reply quickstart-short.md]: i don't have a strong preference so long as we're consistent!

**Pull request closed without merging**